cmd/secrets-config/README.md

```diff
@@ -0,0 +1,179 @@
+% secrets-config-proxy(1) User Manuals secrets-config-proxy(1)
+
+# NAME
+
+secrets-config-proxy – Configure EdgeX API gateway service
+
+# SYNOPSIS
+
+**secrets-config proxy** SUBCOMMAND [OPTIONS]
+
+# DESCRIPTION
+
+Configures the EdgeX API gateway service.
+
+This command is used to configure the TLS certificate for external connections, create authentication tokens for inbound proxy access, and other related utility functions.
+
+Proxy configuration commands (listed below) require access to the secret store master key in order to generate temporary secret store access credentials.
+
+# OPTIONS
+
+  * **--confdir** _/path/to/directory/with/configuration.toml_ (optional)
```

Comment:
If this option optional, or is the configuration file itself optional? If the former, does the tool looks in `./res`?

Reply:
Yes, it says its optional, and behaves like all the existing bootstrap utilities by looking in res by default.

Reply:
Will investigate panic reported at https://github.com/edgexfoundry/edgex-go/pull/2808#pullrequestreview-542153415

Reply:
Panic is fixed.